Gameplay/entities/player/PlayerController.cpp

```diff
@@ -705,6 +705,18 @@ void Hachiko::Scripting::PlayerController::ChangeWeapon(unsigned weapon_idx)
 	}
 }
 
+void Hachiko::Scripting::PlayerController::ReloadAmmo(unsigned ammo)
+{
+	if (0 < ammo && ammo <= MAX_AMMO) 
+	{
+		_ammo_count = ammo;
+		UpdateAmmoUI();
+	}
+	else {
+		HE_LOG("ReloadAmmo: Incorrect AMMO value");
+	}
```

Comment:
Maybe it's a better idea to add an assert statement here instead of a HE_LOG as no one checks there :eyes: